OloEngine/src/OloEngine/Networking/Persistence/WorldPersistenceManager.cpp

```
@@ -0,0 +1,125 @@
+#include "OloEngine/Core/Log.h"
+
+namespace OloEngine
+{
+    void WorldPersistenceManager::Initialize(IWorldDatabase* database, f32 saveIntervalSeconds)
+    {
+        m_Database = database;
+        m_SaveInterval = saveIntervalSeconds;
+        m_TimeSinceLastSave = 0.0f;
+    }
+
+    void WorldPersistenceManager::Shutdown()
```

Cppcheck (2.20.0)

[style] The function 'Shutdown' is never used.

(unusedFunction)